docker/Dockerfile

```diff
@@ -64,7 +65,6 @@ RUN curl -sSL -o openjre-11_linux-x64_bin.tar.gz https://github.com/AdoptOpenJDK
 # this doesn't work here, because docker writes /etc/hosts when it starts the container
 #RUN echo "127.0.0.1 RVSDO OwnerSDO" >> /etc/hosts
 
-USER sdouser
```

Comment:
It is best practice for a container to not run as root. Removing this line will make it run as root when run in docker.